tests/e2e/network/service_annotations.go

```diff
@@ -195,6 +203,37 @@ var _ = Describe("Service with annotation", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ret).To(Equal(true), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
+
+	It("should bound to the specified load balancer from azure-load-balancer-mode [MultipleAgentPools][VMSS]", func() {
+		//get nodelist and providerID specific to an agentnodes
+		nodes, err := utils.GetAgentNodes(cs)
+		Expect(err).NotTo(HaveOccurred())
+		// Get all the vmss names from all node's providerIDs
+		vmss_name := sets.NewString()
+		var resourceGroupName string
+		for _, node := range nodes {
+			providerID := node.Spec.ProviderID
+			matches := scalesetRE.FindStringSubmatch(providerID)
+			if len(matches) != 2 {
+				Skip("Report error")
+			}
+			vmss := matches[1]
+			resourceGroupName = matches[2]
+			vmss_name.Insert(vmss)
+		}
+		if len(strings.TrimSpace(resourceGroupName)) == 0 {
+			Skip("Tests failed")
+		}
+		//Skip if there're less than two vmss nodes
+		if len(vmss_name) < 2 {
+			Skip("at least two vmss are required for azure-load-balancer-mode  validation")
```

Comment:
Skip("azure-load-balancer-mode tests only works for cluster with multiple vmss agent pools")